### DISCUSSION TITLE
GradientBoostingSurvivalAnalysis

### DISCUSSION THREAD

**Discussion by nansy99** [sebp/scikit-survival#253] [Q&A]
Hi! why does Gradient Boosting not offer a _predict_survival_function_ method? 

``gb = GradientBoostingSurvivalAnalysis(learning_rate=1, n_estimators=800, random_state=42, loss='coxph')``
``estimator = gb.fit(X_train, y_train)``
``estimator.predict_survival_function(X_test)``

But I got an error
**AttributeError: 'GradientBoostingSurvivalAnalysis' object has no attribute 'predict_survival_function'**

**Comment by sebp** [comment]
predict_survival_function has been added in version 0.15.0. Are you 
running an older version?
